packages/frontend/src/providers/TrackingProvider.tsx

```diff
@@ -133,13 +133,14 @@ export const TrackingProvider: FC<TrackingData> = ({
         rudder: { writeKey, dataPlaneUrl },
     } = health.data || { rudder: {} };
 
-    const appContext = useMemo(
-        () => ({
-            namespace: 'lightdash',
-            name: LIGHTDASH_APP_NAME,
-            version,
-            build: version,
-        }),
+    const appContext = useMemo<rudderSDK.apiObject>(
+        () =>
+            ({
+                namespace: 'lightdash',
+                name: LIGHTDASH_APP_NAME,
+                version,
+                build: version,
+            } as any as rudderSDK.apiObject),
```

Comment:
Rudder SDK doesn't allow undefined values. Opened a issue in their repo:
https://github.com/rudderlabs/rudder-sdk-js/issues/452